pages/_app.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import App from 'next/app';
+import Head from 'next/head';
+
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { ThemeProvider } from '@material-ui/core/styles';
+
+import theme from 'theme';
+import { Provider as AppContextProvider } from 'AppContext';
+
+export default class DominionApp extends App {
+  componentDidMount() {
+    // Remove the server-side injected CSS.
+    const jssStyles = document.querySelector('#jss-server-side');
+    if (jssStyles) {
+      jssStyles.parentNode.removeChild(jssStyles);
+    }
+  }
+
+  render() {
+    const { Component, pageProps } = this.props;
+
+    return (
+      <>
+        <Head>
+          <title>Dominion: Who Owns the Land</title>
+          <style type="text/css">
+            {`html {
+              fontSize: 14px;
```

Comment:
Should it be 16px ? or is this dominion ? It will affect our rem calculations.

Reply:
Dominion has been using `14px`: https://github.com/CodeForAfrica/Dominion.AFRICA/blob/ef4705315251620d0fe5a96efa4ce92f8cac655d/public/index.html#L25-L29